programs/extcuraengine.hs

```diff
@@ -121,13 +121,16 @@ centeredTrisFromSTL (RectArea (bedX,bedY,_)) stl = shiftedTris
     where
       centerPoint = Point3 (dx,dy,dz)
       shiftedTris = [shiftTri centerPoint tri | tri <- tris] `using` parListChunk (div (length tris) (fromFastℕ threads)) rseq
+      xs = [ xOf.fst <$> triPoints | triPoints <- sidesOf <$> tris ] `using` parListChunk (div (length tris) (fromFastℕ threads)) rseq
+      ys = [ yOf.fst <$> triPoints | triPoints <- sidesOf <$> tris ] `using` parListChunk (div (length tris) (fromFastℕ threads)) rseq
+      zs = [ zOf.fst <$> triPoints | triPoints <- sidesOf <$> tris ] `using` parListChunk (div (length tris) (fromFastℕ threads)) rseq
```

Comment:
Lots of repetition here, maybe refactor common `parListChunk (div (length tris) (fromFastℕ threads)) rseq`.